phsource/ph_arabic

```diff
@@ -15,9 +15,10 @@ phoneme Z // arabic ZAH .Todo
   ipa ðˤ
   lengthmod 5 
   voicingswitch z  
-  Vowelin  f1=1  f2=2700 400 600  f3=200 70 rate len=70
-  Vowelout f1=1  f2=2700 400 600  f3=200 70 rate
-  WAV(vwl_ar/zah) 
+//  Vowelin  f1=1  f2=2700 400 600  f3=200 70 rate len=70
+//  Vowelout f1=1  f2=2700 400 600  f3=200 70 rate
+//  WAV(vwl_ar/zah) // FIXME: This wav file is wrong, therefore English D is used
```

Comment:
Removed lines should be deleted, not commented out. The original will be preserved in the git history.

Reply:
Can I just rewrite history for these fixes?